library/init/meta/interactive.lean

```diff
@@ -6,8 +6,11 @@ Authors: Leonardo de Moura
 prelude
 import init.meta.tactic init.meta.rewrite_tactic init.meta.simp_tactic
 import init.meta.smt.congruence_closure init.category.combinators
+import init.meta.lean_parser
 
 namespace interactive
+@[reducible] meta def parse {α : Type} (p : lean_parser α) : Type := α
```

Comment:
BTW, note that in the example above we use `to_pexpr`. It uses the type class `has_to_pexpr`. It currently used for antiquotations. If we add an instance `has_to_pexpr nat`, then we can write 
```lean
let a := 10 in 
`(%%a + %%a)
```